cypress/integration/features/addons/goToAddons.feature

```diff
@@ -0,0 +1,8 @@
+Feature: To go to addons page on NBC
```

Comment:
It would have been nice if you would have at least answered the questions inside my comments, instead of just resolving all of them...